spec/requests/users_spec.rb

```diff
@@ -71,3 +71,21 @@
     end
   end
 end
+
+describe 'User Requests', type: :request do
+  let(:endpoint) { "/api/users/#{user.id}" }
+  let(:user) { create(:user) }
+  let(:update_params) do
+    {
+      part: 'first'
+    }
+  end
+  describe 'PUT /api/users/:id' do
+    it 'successfully updates a user\'s part' do
+      put endpoint, params: update_params.to_json, headers: authenticated_header(user)
+
+      expect(response).to have_http_status(200)
+      user.part.should eq(User.parts[update_params['part']])
```

Comment:
Just a heads up, `should` shouldn't be used anymore. It was replaced [in 2012](http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/) because of some issues (mentioned in the article I linked).

I think their DNS is down right now, but http://betterspecs.org/ is a good resource for RSpec best practices.

Also, you can consider adding [rubocop-rspec](https://github.com/backus/rubocop-rspec) for spec linting if you want (it'll catch stuff like this)